registrations/serializers.py

```diff
@@ -308,6 +302,68 @@ def validate(self, data):
         return data
 
 
+class SignUpGroupCreateSerializer(serializers.ModelSerializer, CreateSignUpsSerializer):
+    reservation_code = serializers.CharField(write_only=True)
+    attending = serializers.SerializerMethodField(read_only=True)
+    waitlisted = serializers.SerializerMethodField(read_only=True)
+
+    def get_attending(self, obj):
+        attending = obj.signups.filter(attendee_status=SignUp.AttendeeStatus.ATTENDING)
+        return {
+            "count": attending.count(),
+            "people": SignUpSerializer(attending, many=True).data,
+        }
+
+    def get_waitlisted(self, obj):
+        waitlisted = obj.signups.filter(
+            attendee_status=SignUp.AttendeeStatus.WAITING_LIST
+        )
+        return {
+            "count": waitlisted.count(),
+            "people": SignUpSerializer(waitlisted, many=True).data,
+        }
+
+    def create(self, validated_data):
+        validated_data["created_by"] = self.context["request"].user
+        validated_data["last_modified_by"] = self.context["request"].user
+
+        validated_data.pop("reservation_code")
+        reservation = validated_data.pop("reservation")
+        signups_data = validated_data.pop("signups")
+
+        instance = super().create(validated_data)
+
+        for signup_data in signups_data:
+            signup_data["signup_group"] = instance
+            signup_serializer = SignUpSerializer(data=signup_data, context=self.context)
+            signup_serializer.create(signup_data)
+
+        reservation.delete()
+
+        return instance
+
+    class Meta:
+        fields = (
+            "id",
+            "registration",
+            "signups",
+            "reservation_code",
+            "extra_info",
+            "attending",
+            "waitlisted",
```

Comment:
Now `POST /signup_group/` response contains "attending" and "waitlisted" fields, but also "signups" field. I think "attending" and "waitlist" just have duplicate information comparing to "signups" and can removed totally.

Example response at the moment:
```
{
    "id": 2,
    "registration": 2,
    "signups": [
        {
            "id": 102,
            "service_language": null,
            "created_at": "2023-08-28T11:53:32.566984Z",
            "last_modified_at": "2023-08-28T11:53:32.567080Z",
            "created_by": "Jori Lindell - jori.lindell@gmail.com",
            "last_modified_by": "Jori Lindell - jori.lindell@gmail.com",
            "responsible_for_group": true,
            "first_name": "Jori",
            "last_name": "Lindell 1",
            "date_of_birth": "2011-01-01",
            "city": null,
            "email": "jori.lindell@gmail.com",
            "extra_info": null,
            "membership_number": null,
            "phone_number": null,
            "notifications": "none",
            "attendee_status": "attending",
            "street_address": null,
            "zipcode": "00100",
            "presence_status": "not_present",
            "registration": 2,
            "signup_group": 2,
            "native_language": null
        }
    ],
    "extra_info": null,
    "attending": {
        "count": 1,
        "people": [
            {
                "id": 102,
                "service_language": null,
                "created_at": "2023-08-28T11:53:32.566984Z",
                "last_modified_at": "2023-08-28T11:53:32.567080Z",
                "created_by": "Jori Lindell - jori.lindell@gmail.com",
                "last_modified_by": "Jori Lindell - jori.lindell@gmail.com",
                "responsible_for_group": true,
                "first_name": "Jori",
                "last_name": "Lindell 1",
                "date_of_birth": "2011-01-01",
                "city": null,
                "email": "jori.lindell@gmail.com",
                "extra_info": null,
                "membership_number": null,
                "phone_number": null,
                "notifications": "none",
                "attendee_status": "attending",
                "street_address": null,
                "zipcode": "00100",
                "presence_status": "not_present",
                "registration": 2,
                "signup_group": 2,
                "native_language": null
            }
        ]
    },
    "waitlisted": {
        "count": 0,
        "people": []
    }
}
```

Reply:
@jorilindell I've now removed the two fields as suggested in https://github.com/City-of-Helsinki/linkedevents/pull/675/commits/ac5875086bf0548b7efa7cea3deb5437b66a97c9.